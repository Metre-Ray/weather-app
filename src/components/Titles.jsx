import React from 'react';
import styles from './Titles.module.css';

const Titles = () => (
  <div className={styles.titlesContainer}>
    <h1>Weather search</h1>
    <p>Find out weather, temperature and other conditions</p>
  </div>
);

export default Titles;
