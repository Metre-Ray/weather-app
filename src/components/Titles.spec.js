import React from "react";
import { create } from "react-test-renderer";
import Titles from './Titles';

describe("Titles component", () => {
  test("Titles matches the snapshot", () => {
    const titles = create(<Titles />);
    expect(titles.toJSON()).toMatchSnapshot();
  });
});
