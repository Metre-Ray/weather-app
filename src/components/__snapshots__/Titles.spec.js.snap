// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titles component Matches the snapshot 1`] = `
<div>
  <h1>
    Weather finder
  </h1>
  <p>
    Find out weather, temperature and other conditions
  </p>
</div>
`;
